### DISCUSSION TITLE
Should I duplicate my css ?

### DISCUSSION THREAD

**Discussion by YoannBuzenet** [vercel/community#134] [Show & Tell]
Hello,

For most of my components, I use the css variables exported from a `module.scss` file.

But if I want to use these same variable in my global `.css` file, I can't access them. So for now, I defined them again in a css file that I import in it.

Is it good practice though ? Should I define variables in`.module.scss` file AND in a regular `.css` file ? How would you handle it ?

**Comment by MathiasReker** [comment]
To have low coupling, I suggest working around the duplicate solution. Can you send an example of your code?
